src/MultipleSendResult.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Mailer;
+
+use Throwable;
+
+/**
+ * Result of {@see MailerInterface::sendMultiple()}.
+ */
+final class MultipleSendResult
```

Comment:
I'd think about naming a bit:

- `SendResults`
- `Results`

Reply:
But this object for `sendMultiple` only. To call it shortler is OK?

Reply:
Yes. I think `SendResults` is alright.

Reply:
Done